fastlane/Fastfile

```diff
@@ -30,7 +30,7 @@ platform :ios do
   lane :bump do |options|
     new_version_number = options[:version]
     fail ArgumentError, "missing version" unless new_version_number
-    previous_version_number = get_version_number
+    previous_version_number = get_version_number(target: "Purchases")
     increment_version_number(version_number: new_version_number)
```

Comment:
so this increases the version number in both `PurchasesCoreSwift` and `Purchases`??

Reply:
yep! I tried it and it works. 
`increment_version_number` is just a wrapper around Apple's `avgtool`, and it doesn't distinguish between targets, it updates for all targets in the project (including test targets)